buildlogger.proto

```diff
@@ -42,3 +42,15 @@ message LogEndInfo {
   int32 exit_code = 1;
   google.protobuf.Timestamp completed_at = 2;
 }
+
+message BuildloggerResponse {
+  string id = 1;
+  bool success = 2;
+}
+
+
+service Buildlogger {
```

Comment:
should we also have a client-streaming interface?

Reply:
added